src/OmniSharp.Host/Startup.cs

```diff
@@ -130,12 +130,12 @@ public void ConfigureServices(IServiceCollection services)
         {
             Func<RuntimeLibrary, bool> shouldLoad = lib => lib.Dependencies.Any(dep => dep.Name == "OmniSharp.Abstractions" ||
                                                                                        dep.Name == "OmniSharp.Roslyn");
-                       
+
             var assemblies = DependencyContext.Default
                                               .RuntimeLibraries
                                               .Where(shouldLoad)
-                                              .SelectMany(lib => lib.Assemblies)
-                                              .Select(each => loader.Load(each.Name))
+                                              .SelectMany(lib => lib.RuntimeAssemblyGroups.GetDefaultGroup().AssetPaths)
```

Comment:
It seemed to me that they tried deprecating the Assemblies property, but it would be great if you could look into it.